webservice/webservice.py

```diff
@@ -322,7 +326,64 @@ def get_manifest():
     # Get the response back
     logger.info("Creating the API response")
     response = es_td.transform_manifest(filters=filters)
-    # Return the excel file
+    if output_formats == 'tsv':  # returns metadata as TSV file
+        return response
+    elif output_formats == 'bdbag':  # returns presigned URL to a BDBag in S3
+        response_bag = bdbag_response(response)
+        return jsonify(response_bag)
+
+
+def bdbag_response(response_obj):
+    logger = logging.getLogger('dashboardService.webservice.bdbag_response')
+    # Create and return the BDbag folder.
+    bag_name = 'manifest'
+    bag_info = {'organization': 'UCSC Genomics Institute',
+                'data_type': 'TOPMed',
```

Comment:
Why is this hard-coded to TOPMed?

Reply:
This is a left-over from the past. I left them empty now. Perhaps we should give the user the option to fill those before the bag is created in the UI?